prog/kubernetes/kubernetes_cluster_nexus.rb

```diff
@@ -3,27 +3,38 @@
 class Prog::Kubernetes::KubernetesClusterNexus < Prog::Base
   subject_is :kubernetes_cluster
 
-  def self.assemble(name:, version:, private_subnet_id:, project_id:, location:, cp_node_count: 3, target_node_size: "standard-2", target_node_storage_size_gib: nil)
+  def self.assemble(name:, project_id:, location:, version: "v1.32", private_subnet_id: nil, cp_node_count: 3, target_node_size: "standard-2", target_node_storage_size_gib: nil)
     DB.transaction do
       unless (project = Project[project_id])
         fail "No existing project"
       end
 
-      unless ["v1.32"].include?(version)
+      unless ["v1.32", "v1.31"].include?(version)
         fail "Invalid Kubernetes Version"
       end
 
       Validation.validate_kubernetes_name(name)
       Validation.validate_kubernetes_cp_node_count(cp_node_count)
 
-      # TODO: Validate subnet location if given
-      # TODO: Validate subnet size if given
-      # TODO: Create subnet if not given
+      subnet = if private_subnet_id && private_subnet_id != ""
```

Comment:
Do we really need to check for an empty string?

Reply:
You're right, not anymore. I initially crafted the commit to allow empty string as the "default" subnet option in the UI, but later just removed the option altogether.